.go-version

```diff
@@ -1 +1 @@
-1.23.6
+1.24.0
```

Comment:
1.24.1 is out

Reply:
Right, good catch. @Elbehery, can you open a new pull request to bump Go to 1.24.1? Thanks.

Reply:
sure 👍🏽 😍

Reply:
raised https://github.com/etcd-io/bbolt/pull/925